app/stac_api/management/commands/list_asset_uploads.py

```diff
@@ -63,7 +63,7 @@ def are_uploads_equal(s3_upload, db_upload):
                 return False
 
             # Add all db uploads
-            for db_upload in db_uploads_qs:
+            for db_upload in db_uploads_qs:  # pylint: disable=possibly-used-before-assignment
```

Comment:
I would not disable this pylint but fix the code, you can set `db_uploads_qs` to an empty list at the top of the function instead.